.github/workflows/tests.yml

```
@@ -0,0 +1,24 @@
+name: Python package
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # Python 3.6 is not supported by ubuntu-latest
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12', '3.13']
```

Could we remove the comment about '3.6' as well as version '3.7'. The README can be updated to drop saying that we support versions 3.6 and 3.7 as well which I am happy to do after this PR is merged and at the time of release.
```suggestion
        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13']
```